source/geometry/FloatBox2D.ooc

```diff
@@ -40,8 +40,8 @@ FloatBox2D: cover {
 	rightCenter ::= FloatPoint2D new(this right, this center y)
 	topCenter ::= FloatPoint2D new(this center x, this top)
 	bottomCenter ::= FloatPoint2D new(this center x, this bottom)
-	empty ::= this size empty
-	area ::= this size x * this size y
+	isEmpty ::= this size hasZeroArea
```

Comment:
I think David meant a large box, like for keeping stuff inside, which is empty because there is no stuff inside but is still large at the same time.